.github/workflows/build.yaml

```diff
@@ -121,7 +122,8 @@ jobs:
           retention-days: 7
 
       - name: Release
-        uses: qlik-oss/sn-gh-workflows/actions/semantic-release@v1
+        if: ${{ inputs.release }}
+        uses: qlik-oss/sn-gh-workflows/actions/release-it@dvr/release-it
```

Comment:
Not sure how the V1 stuff works, but since this is on a branch I guess it does not work

Reply:
This should be changed to @v1 otherwise the main branch will break. Haven't been able to set it to the same branch that the workflow was triggered from